code/modules/reagents/chemistry_properties/prop_neutral.dm

```diff
@@ -268,12 +268,40 @@
 	rarity = PROPERTY_UNCOMMON
 	category = PROPERTY_TYPE_METABOLITE
 
+/**
+ * For QuickClot removes CHEM_EFFECT_NO_BLEEDING. if above 0C, has no Cryox/Clonex in system. Exception check for cryometabolizing hypothermics
+ *
+ * Affected mob will be evaluated for QuickClot conditions, above 0c, has cryo/clonex no in system and remove the flag regardless of reagent cancel
+ * arguments:
+ * *M - the affected Mob
+ */
+/datum/chem_property/neutral/hypothermic/pre_process(mob/living/M)
+	var/mob/living/carbon/human/effected_human = M
+	///IF are above 0C or no cryo/clonex remove no bleed flag.
+	if ((M.bodytemperature >= T0C || (effected_human.reagents.get_reagent_amount("cryoxadone") == 0 && effected_human.reagents.get_reagent_amount("clonexadone") == 0)) && (effected_human.chem_effect_flags & CHEM_EFFECT_NO_BLEEDING))
+		effected_human.chem_effect_flags &= CHEM_EFFECT_NO_BLEEDING
+
```

Comment:
you dont need to check if the player has the flag before removing it. it just wont do anything.